vcnc-server/vcnc-sampler/app.js

```diff
@@ -85,11 +85,10 @@ function requestHandler(request, response) {
 
 // Create server
 //
-let server;
+// let server;
 function startServer(h, p) {
-  server = http.createServer(requestHandler);
-  server.setTimeout(5000, () => {
-  });
+  const server = http.createServer(requestHandler);
+  server.setTimeout(5000);
```

Comment:
I'm going to approve this, because you really want five seconds.  But I think you're better off with the default.  As I said in another comment, the vda can hold thousands of channels open at the same time.